vllm/worker/model_runner.py

```diff
@@ -789,9 +793,10 @@ def load_model(self) -> None:
                     "This may lead to less accurate results!")
 
         if envs.VLLM_TEST_DYNAMO_GRAPH_CAPTURE:
-            self.model = torch.compile(self.model,
-                                       fullgraph=True,
-                                       backend="eager")
+            self.model = torch.compile(
+                self.model,
+                fullgraph=envs.VLLM_TEST_DYNAMO_FULLGRAPH_CAPTURE,
```

Comment:
can you explain the rationale here?

Reply:
There's a different set of failures when `fullgraph=True`.   Not just graph breaks but actual dynamo errors.  Locally, I am switching between full/not-full constantly to test things.

Reply:
can you send me the specific case where fullgraph fails?

Reply:
There's probably more but these all fail with fullgraph=True and pass with fullgraph=False
```
FAILED tests/models/test_gguf.py::test_models[1-5-32-half-model0] - torch._dynamo.exc.Unsupported: hasattr TensorVariable shard_size
FAILED tests/models/test_gguf.py::test_models[1-5-32-half-model1] - torch._dynamo.exc.InternalTorchDynamoError: 'SymNodeVariable' object has no attribute 'value'
FAILED tests/models/test_gguf.py::test_models[1-5-32-half-model2] - torch._dynamo.exc.Unsupported: hasattr TensorVariable shard_size
FAILED tests/models/test_gguf.py::test_models[1-5-32-half-model3] - torch._dynamo.exc.Unsupported: hasattr TensorVariable shard_size
```